### DISCUSSION TITLE
How to disable the console window when building a dioxus desktop app?

### DISCUSSION THREAD

**Discussion by testforvln** [DioxusLabs/dioxus#1637] [Q&A] [answered]
Hi all, I'm writing a dioxus desktop app on Win10. When I build the app using "dx build --release" and click the exe program, the app always turn up with a backgournd console window. How can I make the console window disapper?

**Comment by ealmloff** [comment, marked as answer]
This was recently added to the bundling instructions here: https://dioxuslabs.com/learn/0.4/cookbook/publishing#preparing-your-application-for-bundling

**Reply by testforvln** [reply]
Thanks!
